workflows/scoring/validate.cwl

```diff
@@ -99,20 +100,43 @@ requirements:
                     return wiki['markdown']
                 else:
                     return None
+            
+            def check_permissions(syn, sub):
+                invalid_reasons = []
+                project_id = sub['entityId']
+
+                try:
+                    perms = syn.getPermissions(project_id, 3385462)
+                    if "READ" not in perms and "DOWNLOAD" not in perms:
+                        invalid_reasons.append("The evaluation panel team cannot read the project.")
```

Comment:
Let's include the full team name here and down below: "Wellcome Data Re-use Prize - Surveillance Evaluation Panel". That way participants can copy paste into the web UI.